foundation/foundation-upgrade/src/test/java/com/boozallen/aissemble/upgrade/migration/v1_7_0/ChartNameMigrationSteps.java

```diff
@@ -0,0 +1,96 @@
+package com.boozallen.aissemble.upgrade.migration.v1_7_0;/*-
```

Comment:
Would add a new line after the `;` so that the `/*-` is on the next line

Reply:
@carter-cundiff good eye! added newline.